src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Printer exists
+        '500':
+          description: >-
+            Erro while adding printer
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              examples:
+                unresolved_hostname:
+                  summary: Unresolved hostname
+                  description:  You are trying to add printer via hostname, but the server was unable to resolve that hostname to an IP.
+                  value:
+                    message: Cannot resolve {hostname} with mDNS
+                invalid_mode:
+                  summary: Wrong CLOUD mode
+                  description: You are trying to add printer in non-cloud mode, but not enough data to identify the printer in network was supplied.
+                  value:
+                    message: Cannot add printer without a token in CLOUD_MODE
+
+  '/organizations/{org_uuid}/printers/{printer_uuid}/connection':
+    post:
+      summary: Change connections state
+      operationId: server.routes.printers.printer_change_connection
+      tags:
+        - printers
+      parameters:
+        - $ref: '#/components/parameters/printer_uuid_param'
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+
+        content:
+          application/json:
+            schema:
+              properties:
+                state:
+                  type: string
+                  enum: [online, offline]
+            examples:
+              connect:
+                value:
+                  state: online
+              disconnect:
+                value:
+                  state: offline
+
+      responses:
+        '204':
+          description: 'OK'
+
+        '400':
+          description: >-
+            Invalid state was sent
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: <state> is an uknown state
+        '500':
+          description: >-
+            Cannot change printers state
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/api_err_response'
+              example:
+                message: Cannon change printer's connection state to <state>
+
+  '/organizations/{org_uuid}/printers/{printer_uuid}/current-job':
+    post:
+      summary: Change  state of current job
+      operationId: server.routes.printers.printer_modify_job
+      tags:
+        - printers
+      parameters:
+        - $ref: '#/components/parameters/printer_uuid_param'
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
```

Comment:
In OpenAPI v3, the `requestBody` is [not required by default](https://swagger.io/docs/specification/describing-request-body/). Since this is what we usually want to be required, we should make sure it's set as `required: true` where applicable. It's definitely missing here but please go through the rest of the definitions and fix this as appropriate.

Reply:
25 occurences, should be all fixed